src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/singlelinejavadoc/Example1.java

```diff
@@ -0,0 +1,42 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="SingleLineJavadoc"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.javadoc.singlelinejavadoc;
+// xdoc section -- start
+public class Example1 {
+  /** @see Math */ // violation
```

Comment:
we have few possible messages from Check https://checkstyle.org/checks/javadoc/singlelinejavadoc.html#Violation_Messages
it is not clear which violation is here, please add message to this line `// violation 'some message'` message is regexp so you can provide only most meanigful part of original message